src/Sign.Cli/CertificateStoreResources.resx

```diff
@@ -121,6 +121,14 @@
     <value>PFX, P7B, or CER file containing a certificate and potentially a private key.</value>
     <comment>{Locked="PFX", "P7B", "CER"} are file extensions.</comment>
   </data>
+  <data name="CertificateFingerprintAlgorithmOptionDescription" xml:space="preserve">
+    <value>SHA fingerprint algorithm. Only accepts SHA-256, SHA-384, or SHA-512</value>
+    <comment>{Locked="SHA-256", "SHA-384", "SHA-512"} are cryptographic algorithms.</comment>
```

Comment:
```suggestion
    <value>Certificate fingerprint algorithm. Allowed values are 'sha256', 'sha384', and 'sha512'.</value>
    <comment>{Locked="sha256", "sha384", "sha512"} are cryptographic hash algorithms.</comment>
```

To be consistent with an existing option, see https://github.com/dotnet/sign/blob/27cf6bdb045d6d4efe04f61895b9b5c5a67b0fdf/src/Sign.Cli/Resources.resx#L138-L141.